.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,49 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔
+
+---
+
+<!-- Thanks for helping grpc-swift! Before you submit your issue, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->
+
+### New Issue Checklist
+
+- [ ] Updated grpc-swift to the latest version
+- [ ] I read the [Contribution Guidelines](https://github.com/grpc/grpc-swift/blob/master/CONTRIBUTING.md)
+- [ ] I searched for [existing GitHub issues](https://github.com/grpc/grpc-swift/issues)
+
+### Issue Description
+<!-- Please include what's happening, expected behavior, and any relevant code samples. 
+Please wrap log statements and code in triple backticks (```) so they get printed correctly on GitHub.
+-->
+
+##### Complete output when running grpc-swift, including the stack trace and command used
+<!-- You can use environment variable for debug 
+e.g. 
+export GRPC_TRACE=all
+export GRPC_VERBOSITY="DEBUG"
+-->
+
+<!-- The output of `GRPC_VERBOSITY` could contain sensitive data, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->
+
+<details>
```

Comment:
How is this rendered on GitHub??

Reply:
it will render the same as [fastlane's one](https://github.com/fastlane/fastlane/issues/new?template=BUG_REPORT.md).  so it only shows `Write` mode.